### DISCUSSION TITLE
SQLite3 DB generated statement with RETURNING keyword did not work

### DISCUSSION THREAD

**Discussion by shivamvarma30** [aarondl/sqlboiler#1037] [Q&A]
Hi, we are using SQlboiler to generate ORM models for SQLite3 db (version 3.37) and it generated the INSERT method with RETURNING keyword in it. However, this doesn't seem to successfully INSERT the data complaining about the fields that are appended with RETURNING. 

SQLite3 supports RETURNING keyword on 3.35 and above and it seems to perfectly accept the insert query with this keyword when done manually. 

Could someone share any suggestions how to resolve this issue? 

Thanks!

**Comment by aarondl** [comment]
There's not much to do but to debug this unfortunately. I'm not an sqlite3 user myself but others seem to be having success with this. Perhaps it's a configuration option when opening the connection to sqlite3?

I'd use `boil.Debug` to see what the actual query being passed to sqlite3 is and verify that the driver you are using to connect to sqlite3 supports returning.
